core/tee/tee_rpmb_fs.c

```diff
@@ -505,6 +512,26 @@ static TEE_Result read_fat(struct rpmb_file_handle *fh, tee_mm_pool_t *p)
 	return res;
 }
 
+#ifdef CFG_ENC_FS
+static TEE_Result generate_fek_and_nonce(struct rpmb_fat_entry *fe)
+{
+	TEE_Result res;
+
+	res = crypto_ops.prng.read(fe->fek, sizeof(fe->fek));
```

Comment:
Should we recall prng in the unprobable (but still possible) case when the resulting fek is all zeros?